wordpress-mobile/WordPress-iOS
# Set images as attachments of posts

Fixes #4942 

To test:

Scenario 1:
 - Start a new post
 - Add a new device media to it (video or image)
 - Publish the post
 - Check on the server if the new media is attached to the new post

Scenario 2:
 - Start a new post
 - Add some content
 - Save the post or Publish
 - Open the post again 
 - Add a new device media to it (video or image)
 - Save the post
 - Check on the server if the new media is attached to the new post

Check on: WordPress.com blogs, jetpack connect blogs and self-hosted blogs,use the visual editor and the legacy editor.

Needs review: @koke
